### DISCUSSION TITLE
Photo note caption

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#14282] [Ideas]
As far as I can tell it is not possible to add text to photo, video and audio notes? I think it would be helpful if it was possible add a caption to a photo note. Just the option to add a few words to help remember why the photo note was made.

**Comment by vshcherb** [comment]
Indeed, it's only possible to rename them. We have further plans to combine audio / video notes with Favorites, so it will make them more customizable.

**Reply by ERYpTION** [reply]
I have only used photo notes so far but find them difficult to manage from the dashboard when I have a lot of them. If an option to customize the name to something more meaningful and perhaps add a few lines of caption was presented right when the image is saved, I think it would help.
